### ISSUE TITLE
Better IO methods for CategoricalValue

### ISSUE THREAD

**Issue by juliohm** [JuliaData/CategoricalArrays.jl#370]
It would be nice to show categories as colored squares with labels so that ML users can see different categories visually in tabular displays such as the one shipped with Pluto:

![image](https://user-images.githubusercontent.com/3345261/135728660-95099c10-74d6-44ca-b7e9-cf486563d275.png)

It suffices to define a prettier IO method for CategoricalValue. Pluto will display it properly in the table.

**Comment by bkamins** [comment]
Or even `Cat(value)` would be nice (and play well with what we print as `eltype` in DataFrames.jl).

**Comment by juliohm** [comment]
Yeah any improvement would be great :) Maybe Colors.jl is a useful dependency here?
